src/test/unit/test_lumigo_utils.py

```diff
@@ -442,6 +444,21 @@ def test_get_edge_host(arg, host, monkeypatch):
     assert get_edge_host("region") == host
 
 
+def test_report_json_extension(monkeypatch, reporter_mock):
+    monkeypatch.setattr(Configuration, "should_report", True)
+    monkeypatch.setenv("LUMIGO_USE_TRACER_EXTENSION", "TRUE")
+    single = [{"a": "b"}]
+    to_send = _create_request_body(single, True).encode()
+    md5str = str(hashlib.md5(to_send).hexdigest())
+    file_name = f"{md5str}_single"
+    file_path = f"/tmp/lumigo-spans/{file_name}"
+    duration = report_json(None, [{"a": "b"}])
+    span_from_file = json.load(open(file_path, "r"))
+    assert path.exists(file_path)
+    assert duration == 0
+    assert span_from_file == single
+
+
```

Comment:
I think it's worth to add a tests where the file exist before running `report_json`